test/indexer/test_hca_indexer.py

```diff
@@ -768,6 +768,56 @@ def test_organoid_priority(self):
         self.assertEqual(inner_cell_suspensions_in_contributions + inner_cell_suspensions_in_aggregates,
                          inner_cell_suspensions)
 
+    def test_field_values_are_sorted(self):
```

Comment:
I think we should write a generic sort validator for all documents, both aggregates and contributions, and plug it into every test method such that every index document from every test bundle is checked. Because the setup for these tests is so time-consuming we simply don't have the luxury to accumulate test cases like this.

You already have the beginnings of such a validator in `_verify_sorted_values`. It just needs to be adapted to handle documents rather than service responses but they are not that dissimilar.